src/php/inject-dependencies.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+Copyright 2015 OCAD University
```

Comment:
I didn't really change the contents of most of the files, so I left the copyrights. For the most part I just moved them.